src/CaptureFileInfo/Manager.cpp

```diff
@@ -77,4 +79,19 @@ void Manager::PurgeNonExistingFiles() {
   SaveCaptureFileInfos();
 }
 
+ErrorMessageOr<void> Manager::FillFromDirectory(const std::filesystem::path& directory) {
+  OUTCOME_TRY(files, orbit_base::ListFilesInDirectory(directory));
+
+  for (const auto& file : files) {
+    if (file.extension() != ".orbit") continue;
+
+    QFileInfo tmp_file_info{QString::fromStdString(file.string())};
+    capture_file_infos_.emplace_back(tmp_file_info.filePath(), tmp_file_info.birthTime());
```

Comment:
What happens when FillFromDirectory has already been called, or a particular file is already present in `capture_file_infos_`. There is no deduplication, right? Maybe this method should clear `capture_file_infos_` before filling it.

Reply:
done